llama_index/indices/managed/google/generativeai/base.py

```diff
@@ -0,0 +1,184 @@
+"""Google GenerativeAI Semantic Vector Store & Attributed Question and Answering.
+
+The GenAI Semantic Retriever API is a managed end to end service that allows
```

Comment:
Nit: end-to-end
https://english.stackexchange.com/questions/531950/end-to-end-vs-end-to-end#:~:text=End%2Dto%2Dend%20is%20an,to%20end%20is%20an%20idiom.